src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -2747,10 +2740,10 @@ private static EvmExceptionType InstructionLog<TTracing>(EvmState vmState, ref E
 
         if (typeof(TTracingInstructions) == typeof(IsTracing))
         {
-            Span<byte> valueToStore = newIsZero ? BytesZero : bytes;
-            bytes = new byte[32]; // do not stackalloc here
-            storageCell.Index.ToBigEndian(bytes);
-            _txTracer.ReportStorageChange(bytes, valueToStore);
+            ReadOnlySpan<byte> valueToStore = newIsZero ? BytesZero.AsSpan() : bytes;
+            byte[] storageBytes = new byte[32]; // do not stackalloc here
```

Comment:
I'm urged to stackalloc

Reply:
https://github.com/NethermindEth/nethermind/pull/937/files#diff-9742ac85df13f5104064b9cf775ea804cc783bc16b9655d45c55774b13be19e6R2001